Source/WebKit/UIProcess/Inspector/mac/WebInspectorUIProxyMac.mm

```diff
@@ -900,15 +900,16 @@ - (IBAction)_popUpButtonAction:(id)sender
     if (!m_isAttached)
         return;
 
-    inspectedViewFrameDidChange(height);
+    inspectedViewFrameDidChange(height + [m_inspectorViewController webView].safeAreaInsets.bottom);
```

Comment:
can you add a comment explaining why `safeAreaInsets.top` isn't used?

Reply:
Ah, I was writing that knowing there won't be any top insets for now. I'll add the top inset anyway as that's more correct and less mysterious.